src/Database/Traits/DeferredBinding.php

```diff
@@ -146,7 +147,11 @@ protected function commitDeferredOfType($sessionKey, $include = null, $exclude =
             $relationObj = $this->$relationName();
 
             if ($binding->is_bind) {
-                $relationObj->add($slaveModel);
+                if ($relationType === 'belongsToMany') {
```

Comment:
```suggestion
                if (in_array($relationType, ['belongsToMany', 'morphToMany'])) {
```

Reply:
May be there must be 'morphedByMany' too?